etna/models/catboost.py

```diff
@@ -10,14 +10,17 @@
 class _CatBoostModel:
     def __init__(
         self,
-        iterations: int = 100,
-        depth: int = 4,
-        learning_rate: float = 0.23,
+        iterations: int = 1000,
+        depth: int = 6,
+        learning_rate: float = None,
         logging_level: str = "Silent",
-        l2_leaf_reg: float = 6.735163225977638,
-        thread_count: int = 4,
+        l2_leaf_reg: float = 3,
+        thread_count: int = -1,
         **kwargs,
     ):
+        if l2_leaf_reg == 3:
```

Comment:
why?

Reply:
Setting this parameter turns off automatic learning_rate selection. See https://catboost.ai/docs/concepts/python-reference_parameters-list.html

Reply:
but default CatboostRegressor's `l2_leaf_reg` value is 3 🤔 
and we try to provide default catboost behavior

Reply:
If you look at the __repr__ of the CatboostRegressor there are None values for all the parameters

Reply:
...but docs say it is 3 🙂 
anyway, this `if` produces confusing behavior: you have no opportunity to set `l2_leaf_reg` with 3. 

if default catboost `l2_leaf_reg` should be None, add None as a default value in our `_CatBoostModel.__init__` method please